### ISSUE TITLE
Presence of null in report

### ISSUE THREAD

**Issue by brokentone** [IanVS/eslint-nibble#135]
While using the tool, I noticed that there can be a report of a "null" rule in the listing:

```
  no-useless-assignment:                                       2|
  no-useless-return:                                          24|          
  null:                                                       25|          
  prefer-arrow-callback:                                       3| 
  prefer-destructuring:                                        9|   
```

The reason is because the `ruleId` from infractions generated by eslint have this definition:
> The rule name that generates this lint message. If this message is generated by the ESLint core rather than rules, this is null.
https://eslint.org/docs/latest/integrate/nodejs-api#-lintmessage-type
So... core violations can be at least parse errors, unused ignore statements, and unused inline configuration -- all 3 of which, you might want to iteratively solve, and the latter 2 can even be auto fixed.

The problem here is that there isn't a corollary way to identify these specific infractions and filter to them. They're going to look like this:
```
        {
          ruleId: null,
          message: "Unused eslint-disable directive (no problems were reported from 'semi').",
          line: 1,
          column: 1,
          severity: 2,
          nodeType: null,
          fix: [Object]
        }
```

The easiest way *might* be for us to modify these objects, parse the message, and invent our own ruleId... maybe we call it "@core/unused-eslint-disable" or something.

Still musing on this, and working on another PR to add some tests to reproduce this issue, but obviously the issue could be reproduced with the following code:

```
/* eslint-disable semi */
var i = 0;
``` 
were the rules `semi: ['error', 'always'],`

**Comment by johannes-lindgren** [comment]
@IanVS, I still see `null` in my reports

**Comment by brokentone** [comment]
This is an open issue... I still need to float an exact suggestion of how to solve this one.
